src/registry/presets/decomposeCustomLabelsBeta.json

```diff
@@ -1,44 +1,32 @@
 {
   "childTypes": {
-    "customlabel": "customlabels"
+    "customlabel": ""
```

Comment:
This feels like hidden magic.  Is it explained what this means anywhere?

Reply:
https://github.com/forcedotcom/source-deploy-retrieve/pull/1392/files#diff-61c4540d4561b7af0f31a1721a2f90d455a9f7f270675fc9feaa024b6ba022a7R175

Reply:
[yeah, I sometimes wish json could have comments!]